### ISSUE TITLE
Cannot write in the composer with Safari on macOS / iOS

### ISSUE THREAD

**Issue by eomannem** [nextcloud/mail#6512]
### Steps to reproduce

Reply an E-Mail, writing in the messsage-filed with MACOSX Desktop Safari is not possible

### Expected behavior

Reply an E-Mail, writing in the messsage-filed with MACOSX Desktop Safari should be possible like with Chrome and Firefox

### Actual behavior

Reply an E-Mail, writing in the messsage-filed with MACOSX Desktop Safari is not possible

**Comment by ChristophWurst** [comment]
I do not understand your report.

Could you rephrase this and write down steps it requires to trigger the bug?

**Comment by davidpr78** [comment]
You just need a Mac or an iPad and Safari Browser. Try to write (type text into the editor) an e-mail.

**Comment by davidpr78** [comment]
Not funny for my collegues 🤨

**Comment by tatrapikao** [comment]
confirmed on IPhone also seems to be the same Problem here: https://github.com/nextcloud/mail/issues/6507

**Comment by marcoambrosini** [comment]
This is currently happening in our company cloud too. When you click into the textarea in the composer, no cursor is displayed and it's not possible to type anything.

**Comment by marcoambrosini** [comment]
Ok I looked into it and you need `-webkit-user-select: text;` in the `<div contenteditable="true">` element

**Comment by GretaD** [comment]
> Great! When will the update be availible?

Hello,

I just released 1.13.5, its also on 1.12.7. Please update :)
